wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md

```diff
@@ -75,9 +75,9 @@ You have these requirements if you failed a Beatmap Nominator application and ap
 
 Users who have recently left the Beatmap Nominators may have different requirements to rejoin compared to when they first joined. For users who have performed well prior to leaving, they may be able to rejoin with relaxed requirements. The NAT will inform the user if they have the opportunity to fast-track their way back into the Beatmap Nominators.
 
-If you were a Beatmap Nominator before, these requirements apply unless you have failed an application since last time you were Beatmap Nominators.
+If you were a Beatmap Nominator before, these requirements apply unless you have failed an application since last time you were Beatmap Nominator.
```

Comment:
```suggestion
If you were a Beatmap Nominator before, these requirements apply unless you have failed an application since last time you were a Beatmap Nominator.
```